lib/happo/public/happo-runner.js

```diff
@@ -221,9 +231,9 @@ window.happo = {
     // Set up the initial object that we will mutate in our recursive function.
     var box = {
       bottom: 0,
-      left: 0,
+      left: undefined,
```

Comment:
Good point, I hadn't thought of that.

On Wednesday, 14 September 2016, Joe Lencioni <notifications@github.com>
wrote:

> In lib/happo/public/happo-runner.js
> <https://github.com/Galooshi/happo/pull/150#discussion_r78686285>:
>
> > @@ -221,9 +231,9 @@ window.happo = {
> >      // Set up the initial object that we will mutate in our recursive function.
> >      var box = {
> >        bottom: 0,
> > -      left: 0,
> > +      left: undefined,
>
> Instead of adding the wrapper, you could probably use Infinity for these
> values.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Galooshi/happo/pull/150/files/3c9f5686d989f5f0af709ab87a9b0450341b838a#r78686285>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAjS5f8TcqlV-3fqNkltjRk_6Lcxyyjyks5qp379gaJpZM4J7Y2e>
> .
>